app/presenters/url_status_presentation.rb

```diff
@@ -3,17 +3,26 @@ module UrlStatusPresentation
 
   def status_description
     return "" unless status
-    return "Good" if status == '200'
-    return "Broken Link #{status}" if status.start_with?('4')
-    return "Server Error #{status}" if status.start_with?('5')
-    status
+    return "Good" if status == "ok"
+    if status == "broken"
+      "Broken: #{link_errors.keys.first}"
+    elsif status == "caution"
+      "Note: #{link_warnings.keys.first}"
+    else
```

Comment:
In my head, I had thought we were ignoring links when they came back as pending, but I'm no longer sure that's the case, would be worth checking.